src/Layer/OGC3DTilesLayer.js

```diff
@@ -0,0 +1,406 @@
+    }
+
+    /**
+     * Sets the lruCache and download and parse queues so they are shared amongst all tilesets.
+     * @private
+     */
+    _setupCacheAndQueues() {
+        if (lruCache === null) {
+            lruCache = this.tilesRenderer.lruCache;
+        } else {
+            this.tilesRenderer.lruCache = lruCache;
+        }
+        if (downloadQueue === null) {
+            downloadQueue = this.tilesRenderer.downloadQueue;
+        } else {
+            this.tilesRenderer.downloadQueue = downloadQueue;
+        }
+        if (parseQueue === null) {
+            parseQueue = this.tilesRenderer.parseQueue;
+        } else {
+            this.tilesRenderer.parseQueue = parseQueue;
+        }
+    }
+
+    /**
+     * Binds 3d-tiles-renderer events to this layer.
+     * @private
+     */
+    _setupEvents() {
+        for (const ev of Object.values(OGC3DTILES_LAYER_EVENTS)) {
+            this.tilesRenderer.addEventListener(ev, (e) => {
+                this.dispatchEvent(e);
+            });
+        }
+    }
+
+    /**
+     * Setup 3D tiles renderer js TilesRenderer with the camera, binds events and start updating. Executed when the
+     * layer has been added to the view.
+     * @param {View} view - the view the layer has been added to.
+     * @private
+     */
+    _setup(view) {
+        this.tilesRenderer.setCamera(view.camera3D);
+        this.tilesRenderer.setResolutionFromRenderer(view.camera3D, view.renderer);
+        // Setup whenReady to be fullfiled when the root tileset has been loaded
+        let rootTilesetLoaded = false;
+        this.tilesRenderer.addEventListener('load-tile-set', () => {
+            view.notifyChange(this);
+            if (!rootTilesetLoaded) {
+                rootTilesetLoaded = true;
+                this._res();
+            }
+        });
+        this.tilesRenderer.addEventListener('load-model', ({ scene }) => {
+            scene.traverse((obj) => {
+                this._assignFinalMaterial(obj);
+                this._assignFinalAttributes(obj);
+            });
+            view.notifyChange(this);
+        });
+        // Start loading tileset and tiles
+        this.tilesRenderer.update();
+    }
+
+    /**
+     * Replace materials from GLTFLoader by our own custom materials. Note that
+     * the replaced materials are not compiled yet and will be disposed by the
+     * GC.
+     * @param {Object3D} model
+     * @private
+     */
+    _assignFinalMaterial(model) {
+        let material = model.material;
+
+        if (model.isPoints) {
+            const pointsMaterial = new PointsMaterial({
+                mode: this.pntsMode,
+                shape: this.pntsShape,
+                classificationScheme: this.classification,
+                sizeMode: this.pntsSizeMode,
+                minAttenuatedSize: this.pntsMinAttenuatedSize,
+                maxAttenuatedSize: this.pntsMaxAttenuatedSize,
+            });
+            pointsMaterial.copy(material);
```

Comment:
This overwrites `pointsMaterial.userData` and thus removes `pointsMaterial.userData.needTransparency = {}` initialized at     `PointsMaterial` construction. This breaks conditional transparency on classified point clouds.
I suggest modifying [this method](https://github.com/iTowns/itowns/blob/cb410cd2f6f7d0562473bd04d1ccd18ecbd42312/src/Renderer/PointsMaterial.js#L284-L300). What do you think? We might add something like that before copying : 
```
const needTransparency = source.userData.needTransparency !== undefined ? source.userdata.needTransparency
        : this.userData.needTransparency;
```
and then apply again the property after copying : 
```
this.userData.needTransparency = needTransparency;
```

Reply:
@jailln @Desplandis

Reply:
good catch, thanks ! I implemented what you suggested